src/test/java/HandleTruthTest.java

```diff
@@ -1,8 +1,76 @@
 import org.junit.Test;
+import java.util.*;
+import static org.junit.Assert.assertEquals;
 
 public class HandleTruthTest {
+
     @Test
     public void makeshiftTest() {
-        HandleTruth.wordCount();
+        String[] testMessages = {
```

Comment:
if your tests are added to the existing makeshiftTest(), consider changing the name to something more formal. Your tests are way better for being name "makeshift" :)